deeplearning4j/deeplearning4j-ui-parent/deeplearning4j-play/src/main/resources/deeplearning4jUiAssets/legacy/bootstrap-theme.min.css

```diff
@@ -1,6 +0,0 @@
-/*!
```

Comment:
This is still in use in `ConvolutionalModule`, here: https://github.com/deeplearning4j/deeplearning4j/blob/cdbd27686941a6029af2992a3fd08f8cd6c980af/deeplearning4j/deeplearning4j-ui-parent/deeplearning4j-play/src/main/views/org/deeplearning4j/ui/views/convolutional/Activations.scala.html#L30

This warning is logged if visiting /activations page:
```
15:12:00.042 [application-akka.actor.default-dispatcher-6] WARN  o.d.ui.play.staticroutes.Assets - Could not find requested UI asset: legacy/bootstrap-theme.min.css
java.io.FileNotFoundException: deeplearning4jUiAssets/legacy/bootstrap-theme.min.css cannot be opened because it does not exist
```

Reply:
Thanks. Issue here: https://github.com/deeplearning4j/deeplearning4j/issues/7217